.travis.yml

```diff
@@ -4,4 +4,4 @@ python:
 install:
   - pip install -r requirements.txt -r requirements-dev.txt
 script:
-  - make unittest
+  - make test
```

Comment:
If both targets work, I'd leave it as "unittest" tbh - explicit better than implicit.
Also, one day functests might come back ;)